doc/tutorials/enable_s5.rst

```diff
@@ -204,72 +271,69 @@ How to Test
 
 #. Refer to the :ref:`enable_s5` section to set up the S5 environment for the User VMs.
 
-   .. note:: RT-Linux's UUID must use ``495ae2e5-2603-4d64-af76-d4bc5a8ec0e5``. Also, the
-      shared EFI image is required for launching the RT-Linux VM.
-
    .. note:: Use the ``systemctl status life_mngr.service`` command to ensure the service is working on the LaaG or RT-Linux:
 
       .. code-block:: console
 
            * life_mngr.service - ACRN lifemngr daemon
-           Loaded: loaded (/usr/lib/systemd/system/life_mngr.service; enabled; vendor p>
-           Active: active (running) since Tue 2019-09-10 07:15:06 UTC; 1min 11s ago
-           Main PID: 840 (life_mngr)
+           Loaded: loaded (/lib/systemd/system/life_mngr.service; enabled; vendor preset: enabled)
+           Active: active (running) since Thu 2021-11-11 12:43:53 CST; 36s ago
+           Main PID: 197397 (life_mngr)
 
    .. note:: For WaaG, we need to close ``windbg`` by using the ``bcdedit /set debug off`` command
       IF you executed the ``bcdedit /set debug on`` when you set up the WaaG, because it occupies the ``COM2``.
 
-#. Use the ``acrnctl stop`` command on the Service VM to trigger S5 to the User VMs:
+#. Use the ``user_vm_shutdown.py`` in the Service VM to shut down the User VMs:
+
+   .. code-block:: none
 
-   .. code-block:: console
+      sudo python3 ~/user_vm_shutdown.py <User VM name>
 
-      # acrnctl stop vm1
+   .. note:: The User VM name is configured in the life_mngr.conf of User VM.
 
 #. Use the ``acrnctl list`` command to check the User VM status.
 
-   .. code-block:: console
+   .. code-block:: none
 
-      # acrnctl list
-      vm1		stopped
+      acrnctl list
```

Comment:
Assuming you need to be root for this (I'm not 100% sure anymore but believe it to be the case), add `sudo`

Reply:
Thanks, you are right, will add sudo.